README.md

```diff
@@ -10,28 +12,18 @@ To execute all pending migrations you can run:
 
     <path-to-bin>/typo3 migrations:migrate
 
-This will give you an output like this:
+# Upgrade from TYPO3 9/10 to 11/12
 
+Moving to TYPO3 11, this extensions switched from doctrine/migration 2.x to 3.x for compatibility
+with PHP 8.
+
+After running the Database Compare migrations, don't forget to run the Upgrade Wizards which will
+migrate the table `migration_statustable`:
```

Comment:
`migration_statustable` is the name of the wizard, not the table, right? This is not really understandable here IMO.

Reply:
True. But I understand it  :)